axiom-eth/src/rlp/mod.rs

```diff
@@ -145,6 +156,57 @@ pub struct RlpArrayTrace<F: ScalarField> {
     // pub array_trace: RlcTrace<F>,
 }
 
+
+#[derive(Clone, Debug)]
+pub struct RlpOfRlpTrace<F: ScalarField> {
+    pub len_trace: RlcTrace<F>,
+    pub item_trace: Vec<RlpItemTrace<F>>,
+    // to save memory we don't need this
+    // pub array_trace: RlcTrace<'v, F>,
+}
+
+
+#[derive(Clone, Debug)]
+pub struct RlpItemWitness<F: ScalarField> { // EDITING HERE ********
```

Comment:
Is this any different from `RlpFieldWitness`? (I do agree `RlpFieldWitness` is a better name)